frontend/src/locales/fr.po

```diff
@@ -738,19 +752,19 @@ msgstr "Les doublons seront ignorés et n'apparaîtront qu'une seule fois."
 msgid "Dutch"
 msgstr "Néerlandais"
 
-#: src/pages/settings/advanced.vue:304
+#: src/pages/settings/advanced.vue:305
 msgid "Dynamic Previews"
 msgstr "Avant-premières non cachées"
 
-#: src/pages/settings/advanced.vue:305
+#: src/pages/settings/advanced.vue:306
 msgid ""
 "Dynamic rendering requires a powerful server. It is not recommended for NAS "
 "devices."
 msgstr ""
 "Le rendu à la demande nécessite un processeur puissant et n'est pas "
 "recommandé pour les petits serveurs domestiques ou les périphériques NAS."
```

Comment:
```suggestion
"Recommandé pour les petits serveurs domestiques ou les périphériques NAS."
```

Reply:
There shouldn't be an uppercase letter in the middle of this sentence.